core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -168,7 +168,7 @@ object ConsoleProducer {
       .withRequiredArg
       .describedAs("request required acks")
       .ofType(classOf[java.lang.String])
-      .defaultsTo("1")
+      .defaultsTo("-1")
```

Comment:
The change makes sense. It looks like it was missed in the KIP. Perhaps it is worthwhile mentioning the change on the vote thread?

Reply:
Good suggestion! Sent a note in the KIP vote thread. Thanks.